### PR TITLE
Restore buyer and supplier fields

### PR DESCRIPTION
**Pull request by duncandewhurst**
Related to https://github.com/open-contracting-extensions/public-private-partnerships/issues/236

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,13 +17,13 @@ The consolidated OCDS for PPPs extension can be declared in [OCDS package metada
 }
 ```
 
-## About PPP extensions 
+## About PPP extensions
 
 OCDS for PPPs is constructed from [a number of different modular extensions to OCDS](http://standard.open-contracting.org/profiles/ppp/latest/en/extensions/), most of which can be used independently from the profile.
 
-This repository contains one additional extensions that forms part of the OCDS for PPPs profile. This extension introduces a number of fields and building blocks that are specific to PPP disclosure against the World Bank Framework, as well as a set of modifications that only apply in an OCDS Profile (e.g. removing standard fields that are not required in the profile). 
+This repository contains one additional extensions that forms part of the OCDS for PPPs profile. This extension introduces a number of fields and building blocks that are specific to PPP disclosure against the World Bank Framework, as well as a set of modifications that only apply in an OCDS Profile (e.g. changing closed codelists).
```
Comment (jpmckinney) [inline README.md]: If https://github.com/open-contracting-extensions/public-private-partnerships/issues/237 is closed, then we can remove the last clause entirely.

```suggestion
This repository contains one additional extension that forms part of the OCDS for PPPs profile. This extension introduces a number of fields and building blocks that are specific to PPP disclosure against the World Bank Framework, as well as a set of modifications that only apply in an OCDS Profile (e.g. changing closed codelists).
```

**Comment by duncandewhurst** [comment]
Tests are failing because the changelog mentions `awards/suppliers` which doesn't appear in `release-schema.json`.

> If open-contracting-extensions/public-private-partnerships#237 is closed, then we can remove the last clause entirely.

I'll make this change in a separate PR for #237

**Comment by jpmckinney** [comment]
> Tests are failing because the changelog mentions `awards/suppliers` which doesn't appear in `release-schema.json`.

Dot notation needs to be used (awards.suppliers).

**Comment by jpmckinney** [comment]
Without whitespace changes: https://github.com/open-contracting-extensions/ocds_ppp_extension/pull/9/files?w=1

**Pull request merged**